demosplan/DemosPlanCoreBundle/Event/Procedure/PostProcedureUpdatedEvent.php

```diff
@@ -12,21 +12,66 @@
 
 namespace demosplan\DemosPlanCoreBundle\Event\Procedure;
 
+use DemosEurope\DemosplanAddon\Contracts\Entities\EntityInterface;
 use DemosEurope\DemosplanAddon\Contracts\Events\PostProcedureUpdatedEventInterface;
 use demosplan\DemosPlanCoreBundle\Entity\Procedure\Procedure;
 use demosplan\DemosPlanCoreBundle\Event\DPlanEvent;
+use ReflectionClass;
 
 class PostProcedureUpdatedEvent extends DPlanEvent implements PostProcedureUpdatedEventInterface
 {
-    protected Procedure $procedure;
+    protected Procedure $procedureBeforeUpdate;
+    protected Procedure $procedureAfterUpdate;
 
-    public function __construct(Procedure $procedure)
+    public function __construct(Procedure $procedureBeforeUpdate, Procedure $procedureAfterUpdate)
     {
-        $this->procedure = $procedure;
+        $this->procedureBeforeUpdate = $procedureBeforeUpdate;
+        $this->procedureAfterUpdate = $procedureAfterUpdate;
     }
 
-    public function getProcedure(): Procedure
+    public function getProcedureBeforeUpdate(): Procedure
     {
-        return $this->procedure;
+        return $this->procedureBeforeUpdate;
+    }
+
+    public function getProcedureAfterUpdate(): Procedure
+    {
+        return $this->procedureAfterUpdate;
+    }
+
+    public function getModifiedValues(): array
```

Comment:
Typehint missing.

Reply:
added in [2426b23](https://github.com/demos-europe/demosplan-core/pull/1741/commits/2426b238e6a21626369a0c2acd1901a00afa6995)